dashboards-reports/opensearch_dashboards.json

```diff
@@ -1,6 +1,6 @@
 {
   "id": "reportsDashboards",
-  "version": "1.3.0.0",
+  "version": "1.3.1.0",
   "opensearchDashboardsVersion": "1.3.0",
```

Comment:
dashboards core not 1.3.1?

Reply:
No just opensearch component

Reply:
then this PR should not bump reportsDashboards?